server/server.go

```diff
@@ -475,3 +475,12 @@ func (p *loggingPlatform) Version() (v string, err error) {
 	}()
 	return p.platform.Version()
 }
+
+func (p *loggingPlatform) Export(ids []flux.ServiceID) (config string, err error) {
+	defer func() {
+		if err != nil {
+			p.logger.Log("method", "Export", "error", err, "config", config)
```

Comment:
`config` could be pretty large here, I'm not sure if we really want to log it even on error.